real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java

```diff
@@ -60,34 +60,33 @@
 	private double instVelocityLast = 0.0;
 	private Date timeLast = new Date();
 	
-	Publisher messagePub_enc;
-	Publisher messagePub_pot;
-	Publisher messagePub_controllerSteering;
-	Publisher brakePub; 
+	private Publisher messagePub_enc;
+	private Publisher messagePub_pot;
+	private Publisher messagePub_controllerSteering;
+	private Publisher brakePub; 
 	
-	Publisher statePub_enc;
-	Publisher statePub_pot;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Perhaps 'statePub_enc' could be replaced by a local variable.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1770851711/issues/source?bid=2938665&fileBranchId=3059924#l68)